### PR TITLE
the butchering

### PR DESCRIPTION
**Pull request by antonbobkov**

### CODE REVIEW

File: p2pspec.md [hunk 1]
```diff
@@ -1,6 +1,14 @@
...
+::: POTATO MESSAGES :::
+
+::establishing connection::
+
+Handshake on connection.
+Not necessary?
+Every message will be signed as is.
+Handshake may be useful to prevent TCP connection spam,
+and possible replay attacks for some messages.
+Handshake is stateful.
```
Comment (pdlla) [inline p2pspec.md]: state 😱

File: p2pspec.md [hunk 2]
```diff
@@ -11,23 +19,30 @@
...
 
-:potato TCP msg:
-- TCP nonsense built over UDP
-  - prob easier just to use TCP...
+:: Player to Verifier::
 
-:txs:
-- A->B send tx (that's it)
+- send_transactions [Transaction]
+  - sends a list of transactions from a player to a verifier
```
Comment (pdlla) [inline p2pspec.md]: :o

Comment (pdlla) [reply p2pspec.md]: feature creep

Comment (antonbobkov) [reply p2pspec.md]: it is? How would a verifier get a transactoion in the first place?

Comment (pdlla) [reply p2pspec.md]: the feature creep is that you changed it to sending a list of transactions instead of 1 at a time 🙀

Comment (pdlla) [reply p2pspec.md]: 🙀

Comment (pdlla) [reply p2pspec.md]: 🙀

Comment (pdlla) [reply p2pspec.md]: 🙀

Comment (pdlla) [reply p2pspec.md]: 🙀

Comment (pdlla) [reply p2pspec.md]: [🙀 🙀 🙀 🙀 🙀 🙀 🙀 🙀 🙀 🙀 🙀 🙀 🙀 🙀 🙀 🙀 🙀 🙀 🙀 🙀 🙀 ]

Comment (antonbobkov) [reply p2pspec.md]: can send a bunch of signleton messages 😱 
just didn't wanna rewrite it in case we do TCP after all and decide to bundle those

Comment (antonbobkov) [reply p2pspec.md]: [🙀, 🙀, 🙀, 🙀, 🙀, 🙀]

Comment (pdlla) [reply p2pspec.md]: what about
[[🙀],[🙀],[🙀],[🙀],[🙀],[🙀],[🙀],[🙀],[🙀],[🙀],[🙀],[🙀]]?

**Pull request merged**